keyboards/ymdk/bface/keymaps/via/rules.mk

```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = no
```

Comment:
```suggestion
```

Reply:
When I don't include those two I get the following error message: `error "Enabling Mousekeys/Extrakeys and Raw HID at the same time is not currently supported on V-USB."`
With them my keymap seems to work in VIA.

Reply:
I'm not sure how that's possible given your fork appears to be up to date - enabling mouse/extra with VIA is in fact now supported - and this PR has been opened from the master branch (which, as a side note, is [not really recommended](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)).

Reply:
I updated  my fork and indeed it now works. Thanks for the advice!

Reply:
So could you remove them from the rules.mk?

Reply:
Oh oops! Done.